clar2wasm/src/wasm_generator.rs

```diff
@@ -1663,7 +1663,7 @@ impl WasmGenerator {
                     builder.call(
                         self.module
                             .funcs
-                            .by_name("stdlib.burn_block_height")
+                            .by_name("burn_block_height")
```

Comment:
Sames as above. `stdlib.` prefix is now part of function present on Clarity standard library.